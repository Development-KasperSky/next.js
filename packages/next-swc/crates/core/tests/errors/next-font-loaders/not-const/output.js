import inter1 from '@next/font/google?Inter;{"variant":"400"}';
import inter2 from '@next/font/google?Inter;{"variant":"400"}';
var i = 10;
var i2 = 20;
