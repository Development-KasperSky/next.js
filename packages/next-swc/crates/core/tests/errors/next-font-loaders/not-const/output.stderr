
  x Font loader calls must be assigned to a const
   ,-[input.js:4:1]
 4 | ,-> var inter1 = Inter({
 5 | |     variant: '400',
 6 | `-> })
   `----

  x Font loader calls must be assigned to a const
    ,-[input.js:9:1]
  9 | ,-> let inter2 = Inter({
 10 | |     variant: '400',
 11 | `-> })
    `----
